chapters/results/sections/non_euclidean/subsections/spherical.tex

```diff
@@ -28,7 +28,7 @@ \subsection{Spherical space}
 Furthermore, unlike in Euclidean geometry, objects further from the player do not always appear smaller, sometimes they can even appear larger.
 This effect of "reversed perspective" can be seen in both games as shown in \autoref{fig:spherical-space-reversed-perspective}.
 In \autoref{fig:spherical-space-reversed-perspective-hyper}, the car and the bot to the right of the car's hood are further from the camera than other objects in the scene.
-\begin{figure*}[h]
+\begin{figure*}[!htb]
```

Comment:
why is this figure*?

Reply:
There are to may to comment so I'll just ask here. Why are some figure* and some figure?

Reply:
Honestly idk I copy-pasted this from latex stack exchange

Reply:
![image](https://github.com/Non-Euclidean-World/Thesis/assets/54417806/38b062f4-3c1e-428e-a717-1addd2cc5551)

Reply:
but when i change it it seems like it doesnt matter, works either way

Reply:
should I change this to `figure`?